src/Formats.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core;
+
+class Formats
+{
+    public const DATETIME = "datetime:Y-m-d h:i:s";
```

Comment:
```suggestion
    public const DATETIME = "Y-m-d h:i:s";
```
That will be enough.